packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -105,6 +113,8 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
     getLogger: monitor((name: string) => customLoggers[name]),
 
     getInitConfiguration: monitor(() => getInitConfigurationStrategy()),
+
+    getInternalContext: monitor((startTime?: StartTime) => getInternalContextStrategy(startTime)),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** unless there is a reason to promote it, ‏what about removing `StartTime` type?
```suggestion
    getInternalContext: monitor((startTime?: number | undefined) => getInternalContextStrategy(startTime)),
```